Adam/cnx 1217 arcgis toc not unblocked after cancellation during

adamhathcock opened the pull request:
Change ArcGIS is use DefaultThreadContext and QueuedTask where necessary (data/model access)

clairekuang commented:
Can you add this to the code documentation, since arcgis trheading is tricky and we need to make sure we understand the app limitations? https://github.com/Esri/arcgis-pro-sdk/wiki/ProConcepts-Framework#using-queuedtask

clairekuang reviewed (changes requested):
Also don't forget to use our pr template! for pr names and to indicate testing

adamhathcock commented:
I updated https://www.notion.so/speckle/The-Big-Host-App-Threading-Page-14ab78fc7aa6807ca052c6c0869530d2

With the QueuedTask link and updated notes
